packages/edit-site/src/components/layout/style.scss

```diff
@@ -313,5 +313,14 @@
 	.edit-site-layout__header {
 		width: 100%;
 	}
+}
 
+.edit-site-page-template-parts,
+.edit-site-page-templates {
+	h2 {
+		font-size: 0.9rem;
+	}
+	h3 {
+		font-size: 0.82rem;
```

Comment:
These values seem a bit too specific, where they feel a bit arbitrary. I'd defer to @WordPress/gutenberg-design for that 😄

Reply:
Thanks for looking! I was trying to match what was there before, which, if I had to give an opinion, were a tad too small anyway.

Reply:
To my understanding, based on how the code currently works, a way to render a H2 and still get the H5 default font size would be this:

`<Heading as="h2" level={ 5 }>`

In this case, the `level` prop would only determine the font size.
I find this pretty confusing, at the point I'd consider to reinstate the `size` prop on the Heading component. Cc @ciampo 

IMHO, a way more intuitive syntax would be:

`<Heading level={ 2 } size={ 5 >`

To my understanding,  would also better follow the logic in [getHeadingFontSize](https://github.com/WordPress/gutenberg/blob/5ba748472b3e481e89ec3ac4c4e215031cc1eb3c/packages/components/src/ui/utils/font-size.ts#L62-L69), which checks whether the passed value is included or not within the 1-6 values. 

In other words, `level` should only allow values between 1 and 6 and map to the default heading font sizes. Instead, `size` should allow any arbitrary font size value.

Right now, it is possible to pass something like `level={ 48 }` and actually get a 48px font size. That seems misleading to me, as the prop name `level` clearly hints at the headings 1-6 levels.

Will create a separate issue.

Reply:
> <Heading level={ 2 } size={ 5 >

That makes sense, thanks. I can play around with it.

Reply:
+1, we should try to avoid custom sizing wherever possible.

Reply:
I rushed the first commit. I just changed the `<Heading />` props, removed the CSS and I think it's good for another check.